app/controllers/form_controller.rb

```diff
@@ -113,7 +113,11 @@ def find_resource
   end
 
   def find_resource_by_named_id
-    @lettings_log = current_user.lettings_logs.find_by(id: params[:lettings_log_id])
+    if request.path.include? "new"
```

Comment:
It's quite unusual to do this in Rails. Is there anything that stops us from returning a new object if we can't find a letting log?

Reply:
There actually is. We want to differentiate between:
1. found lettings log
2. not found log ( render 404 ) 
3. not found log - but arriving here from the new page